gremlin-go/driver/result.go

```diff
@@ -27,17 +27,17 @@ import (
 
 // Result Struct to abstract the Result and provide functions to use it.
 type Result struct {
-	result interface{}
+	Result interface{}
```

Comment:
One minor item, to avoid confusion with the struct and field names being the same, I'd suggest to change the field name to `Data` instead. 
```suggestion
	Data interface{}
```
So we'll have `r.Result` changed to `r.Data`. 
The rest LGTM. Thanks!

Reply:
done! Thanks 👍